ui/packages/consul-ui/mock-api/v1/internal/ui/service-topology/_

```diff
@@ -52,10 +52,26 @@ ${
     }
     fake.seed(index);
 
+
+    // Randomly make permissive intentions
+    let defaultAllow = false;
```

Comment:
Here I'm trying to create a 33% chance of false/false, false/true, and true/false.

Reply:
If its an equal split probability wise, why not just `({min: 1, max: 3})`, or I think you could also use

```javascript
.randomize([
  {
   defaultAllow: true,
   wildcardIntention: false
  }, {
   defaultAllow: false,
   wildcardIntention: true
  }, {
   defaultAllow: ...,
   wildcardIntention: ...
  }
])
```
if you like that better?

Is there anyway we can set up the mock data so we are guaranteed to get one of every type by default, right now I can see allow and deny but no undefined.